src/simulation/m_start_up.fpp

```diff
@@ -357,7 +360,7 @@ contains
             end if
         end do
 
-        if ((bubbles .eqv. .true.) .or. (hypoelasticity .eqv. .true.)) then
+        if ((bubbles .eqv. .true.) .or. (elasticity .eqv. .true.)) then
```

Comment:
Would be great to be consistent `if ( bubbles .or. elasticity ) then` just like line 626.